src/pjpersist/serialize.py

```diff
@@ -342,6 +355,11 @@ def get_state(self, obj, pobj=None):
             return {'_py_type': 'datetime.datetime',
                     'value': obj.strftime(FMT_DATETIME)}
 
+        # dunno... check type by type or stick with subclasses of MappingView?
```

Comment:
This is not a good comment to leave in the code. State confidently the case you are covering.